src/js/components/stock-movement/modals/RejectRequestModalContent.jsx

```diff
@@ -0,0 +1,121 @@
+import React, { useCallback } from 'react';
+
+import _ from 'lodash';
+import PropTypes from 'prop-types';
+import { Form } from 'react-final-form';
+import { connect } from 'react-redux';
+
+import SelectField from 'components/form-elements/SelectField';
+import TextareaField from 'components/form-elements/TextareaField';
+import RejectRequestModalButtons from 'components/stock-movement/modals/RejectRequestModalButtons';
+import { renderFormField } from 'utils/form-utils';
+import { debouncePeopleFetch } from 'utils/option-utils';
+
+const FIELDS = {
+  recipient: {
+    type: SelectField,
+    label: 'react.rejectRequestModal.to.label',
+    defaultMessage: 'To',
+    attributes: {
+      async: true,
+      valueKey: 'id',
+      labelKey: 'name',
+      className: 'request-modal-select',
+      options: [],
+      filterOptions: options => options,
+    },
+    getDynamicAttr: ({ debouncedPeopleFetch }) => ({
+      loadOptions: debouncedPeopleFetch,
+    }),
+  },
+  sender: {
+    type: SelectField,
+    label: 'react.rejectRequestModal.from.label',
+    defaultMessage: 'From',
+    attributes: {
+      className: 'request-modal-select',
+      disabled: true,
+    },
+  },
+  comment: {
+    type: TextareaField,
+    label: 'react.rejectRequestModal.comment.label',
+    defaultMessage: 'Comment',
+    attributes: {
+      className: 'request-modal-textarea',
+      required: true,
+    },
+  },
+};
+
+const RejectRequestModalContent = ({
+  currentUser,
+  requestor,
+  debounceTime,
+  minSearchLength,
+  closeRejectionModal,
+  rejectRequest,
+}) => {
+  const initialValues = {
+    sender: { id: currentUser?.id, label: currentUser?.name },
+    recipient: { id: requestor?.id, label: requestor?.name },
```

Comment:
I don't mind it, but personally I don't like having all properties in one line, so if it were up to me, I'd do it like:
```js
sender: {
  id: currentUser?.id,
  label: currentUser?.name,
}
```
etc.

We should set this rule on the ESLint I believe, not to mix the conventions.